lte/gateway/docker/README.md

```diff
@@ -41,6 +41,7 @@ containerized AGW by running the following steps inside the VM:
 cd $MAGMA_ROOT/lte/gateway && make run  # You can skip this if you have built the AGW with make before
 for component in redis nghttpx td-agent-bit; do cp "${MAGMA_ROOT}"/{orc8r,lte}/gateway/configs/templates/${component}.conf.template; done
 sudo systemctl stop 'magma@*' 'sctpd' # We don't want the systemd-based AGW to run when we start the containerized AGW
+sudo systemctl start magma_dp@envoy
```

Comment:
Why is this necessary, when we stop all other services and actually want to have it running as dockerized service and not as systemd service?

Reply:
This PR dockerizes the envoy controller, not envoy itself (see the diffs in the two .service files above)

Reply:
I see. Then maybe more a questions of understanding:
It looks to me, if all other Magma services are stopped for the dockerized AGW with `sudo systemctl stop 'magma@*' 'sctpd'`.
Is the envoy service than the only Magma service, which still needs to run on the host for the dockerized AGW?